GOVERNANCE.md

```diff
@@ -0,0 +1,66 @@
+# Governance
+
+## Project Roles
+
+### Contributors
+
+Individuals who want to contribute ideas to the TUF specification. Cleanups and
+clarifications are discussed in [GitHub Issues](
+https://github.com/theupdateframework/specification/issues)
+and submitted as [GitHub Pull Requests](
+https://github.com/theupdateframework/specification/pulls).
+
+New ideas and enhancements to the specification are submitted through the [TUF
+Augmentation Proposal (TAP) process](
+https://github.com/theupdateframework/taps/blob/master/tap1.md).
+
+### Consensus Builder
+
+Ultimate authority for changes to the TUF specification, including changes
+proposed through the [TAP process](
+https://github.com/theupdateframework/taps/blob/master/tap1.md), falls to the
+specification's Consensus Builder.
+
+### TAP Editors
+
+The TAP Editors are a team of core contributors to the TUF project who are
+responsible for reviewing and approving, or rejecting, any proposed
+[TAPs](https://github.com/theupdateframework/taps) and changes to the
+specification.
+
+
+## Change Review Process
+
+__All changes must be submitted as a GitHub Pull Request (PR)__
+
+The submitter of the PR is responsible for responding to feedback from
+reviewers and maintainers. While the PR remains open the submitter is also
+responsible for ensuring the change is in a state which can be merged.
```

Comment:
It might be good to clarify what 'state which can be merged' entails.

Reply:
I made an attempt at clarifying this without adding too much detail in 2844beb